package.json

```diff
@@ -77,8 +78,8 @@
   "dependencies": {
     "@babel/preset-env": "^7.0.0",
     "@quilted/react-auto-headings": "^0.1.0",
-    "@shopify/argo-checkout": "^0.4.0",
-    "@shopify/css-utilities": "^1.0.6",
+    "@shopify/argo-checkout": "^0.7.0",
+    "@shopify/css-utilities": "1.0.5",
```

Comment:
Is this intentional?  Version has dropped and carat removed.

Reply:
Yeah, that's what's being used by checkout-web now.

Reply:
👍